README.md

```diff
@@ -110,25 +113,24 @@ to `False`.
 
 &dagger; Algorithm was an entry to the [2016 NAB Competition](http://numenta.com/blog/2016/08/10/numenta-anomaly-benchmark-nab-competition-2016-winners/).
 
-Installing NAB 1.0
-------------------
+Installing NAB
+--------------
 
 ### Supported Platforms
 
 - OSX 10.9 and higher
 - Amazon Linux (via AMI)
 
-Other platforms may work but have not been tested.
-
+Other platforms may work. NAB has been tested on Windows 10 but is not
+officially supported.
 
 ### Initial requirements
 
 You need to manually install the following:
 
-- [Python 2.7](https://www.python.org/download/)
+- [Python 3](https://www.python.org/download/)
```

Comment:
We may want to specify a minimum version of 3.x based on what we tested on? I know there have been some significant differences.